tests/server_test.inc

```diff
@@ -16,29 +16,14 @@ function server_start_one($host, $port, $code = 'echo "Hello world";', $php_opts
 	global $doc_root, $router, $handles, $ports;
 
 	$php_executable = getenv('TEST_PHP_EXECUTABLE');
+	$php_args = getenv('TEST_PHP_ARGS');
 
 	$descriptorspec = array(
 		0 => STDIN,
 		1 => STDOUT,
 		2 => STDERR,
 	);
 
-	$is_windows = substr(PHP_OS, 0, 3) == 'WIN';
```

Comment:
Maybe fall back to the original implementation when TEST_PHP_ARGS is empty instead? The current approach at least avoids complaints about the typical `phpize; ./configure; make test` not working

With this PR, if I run `make test`, all the server tests will fail because I'm not setting TEST_PHP_ARGS

```
=====================================================================
FAILED TEST SUMMARY
---------------------------------------------------------------------
APC: Bug #61742 preload_path does not work due to incorrect string length (variant 1) [tests/apc54_014.phpt]
APC: Bug #59938 APCIterator fails with large user cache [tests/apc54_018.phpt]
APC: GH Bug #176 preload_path segfaults with bad data [tests/ghbug176.phpt]
```

Reply:
done (with igbinary first)